.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -31,9 +30,6 @@ inputs:
   templateFilePath:
     description: "The path to the template file to use for deployment"
     required: true
-  location:
```

Comment:
To generalize the action, I'd suggest to keep the input here and rename it to `deploymentMetadataLocation`, set the default value in the action and pass it to the PS script

Reply:
Hey @eriqua , you know that's the interesting thing which I don't understand (but maybe you know).
- The input is not used in the current solution (ok whatever, we can change that)
- BUT - it is required. Yet it the parent template does not pass it thru. Any idea how this is possible?

Aside from that - when we talked last time, I think we said it would probably be better to have the default specified rather in a PS script (e.g. `New-TemplateDeployment`) rather than in one of the YAML templates. If we implement what you suggest, we would do the opposite (if we set a default). What do we want to do? :D

Reply:
Note: I updated the scripts with your suggestion (and added some logic for the RG handling)

Reply:
Maybe I don't recall correctly. I'd consider both actions and scripts to be as generic as possible, especially in view of publishing them as public actions (if ever). 
Probably the most correct implementation in that sense would be to remove default from both the script and the action and place it in the reusable template workflow instead?

Reply:
Hey @eriqua, we can do that for sure. Can't say I have a strong opinion on that and the decision was purely based on my take on our last conversation (which may have been incorrect).
Anyhow - we can move it again in the 'master yaml template' no problem. I'll update the template and will re-request a review.

Reply:
@eriqua, it's done